src/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java

```diff
@@ -601,7 +601,7 @@ public boolean onRightClick(ItemUseEvent e, Player p, ItemStack item) {
 			@Override
 			public boolean onRightClick(ItemUseEvent e, Player p, ItemStack item) {
 				if (SlimefunManager.isItemSimiliar(item, SlimefunItems.MAGIC_EYE_OF_ENDER, true)) {
-					e.setCancelled(true);
+					e.getParentEvent().setCancelled(true);
```

Comment:
This fix is not related to this PR, therefore it shouldn't have been pushed here. However, as this is already done, let's keep it.